lib/resources/kernel_module.rb

```diff
@@ -37,7 +38,7 @@ def loaded?
     end
 
     def version
-      if inspec.os[:name] == 'centos' && inspec.os[:release].to_i == 5
+      if inspec.os[:family] == 'redhat'
```

Comment:
This would resolve to `true` it for any version of any rhel-based distro, so high chance it will either break for some of existing versions or it will break for some of future versions. I would suggest to keep this check only for centos v5, to avoid possible regression.